recipe/meta.yaml

```diff
@@ -10,8 +10,8 @@ source:
   sha256: 6b4763f042abc59d299d9c84d97efe077930167dd0d6a5b171a8ba843779f93a
 
 build:
-  number: 1
-  skip: true  # [py<310 or linux]
+  number: 2
+  skip: true  # [py<310]
```

Comment:
- linux gave warnings when wxpython 4.2.1 or 4.2.0 used.
- now that 4.2.2. is released, let's run linux as well.